guides/v3.5.0/models/index.md

```diff
@@ -174,39 +178,6 @@ for your app's models. Both routes and their corresponding controllers have acce
 this shared store; when they need to display or modify a model, they
 first ask the store for it.
 
-## Convention Over Configuration with JSON API
```

Comment:
@jessica-jordan 

I think this part is useful too. In fact, the content you mentioned in this comment has been moved to the `Ember Flexibility` section in this PR like:

> By default, Ember Data is designed to work out of the box with [JSON API][json-api]
JSON API is a formal specification for building conventional, robust, and performant
APIs that allow clients and servers to communicate model data.

We can extend and/or improve this or any other part, I didn't try more changes because I was not sure how to complete it.

Reply:
I see now! [From what I got from this discussion](https://discuss.emberjs.com/t/when-to-add-ember-data/15050/14) it indeed makes sense to  reduce the content of the `Convention over Configuration` section, but I think leaving in some more of the prose on the benefits would be useful. I think this following bit is still crucial and could be re-added to the bit about jsonapi that's already part of the `Ember Data Flexibility` seciton

```
JSON API standardizes how JavaScript applications talk to servers, so	
you decrease the coupling between your frontend and backend, and have	
more freedom to change pieces of your stack.
```